src/main/java/uk/gov/hmcts/darts/arm/service/impl/BatchCleanupArmResponseFilesServiceCommon.java

```diff
@@ -34,42 +33,33 @@
 import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.ARM_RESPONSE_PROCESSING_FAILED;
 import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.ARM_RPO_PENDING;
 import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.STORED;
-import static uk.gov.hmcts.darts.task.api.AutomatedTaskName.BATCH_CLEANUP_ARM_RESPONSE_FILES_TASK_NAME;
 
-@Service
 @RequiredArgsConstructor
 @Slf4j
-public class BatchCleanupArmResponseFilesServiceImpl implements BatchCleanupArmResponseFilesService {
-
-    private static final String taskName = BATCH_CLEANUP_ARM_RESPONSE_FILES_TASK_NAME.getTaskName();
+public class BatchCleanupArmResponseFilesServiceCommon implements BatchCleanupArmResponseFilesService {
 
     private final ExternalObjectDirectoryRepository externalObjectDirectoryRepository;
-
     private final ObjectRecordStatusRepository objectRecordStatusRepository;
     private final ExternalLocationTypeRepository externalLocationTypeRepository;
-
     private final ArmDataManagementApi armDataManagementApi;
     private final UserIdentity userIdentity;
     private final ArmBatchCleanupConfiguration batchCleanupConfiguration;
     private final ArmDataManagementConfiguration armDataManagementConfiguration;
     private final CurrentTimeHelper currentTimeHelper;
     private final ArmResponseFileHelper armResponseFileHelper;
-
-    private UserAccountEntity userAccount;
+    private final String manifestFilePrefix;
 
 
     @Override
     public void cleanupResponseFiles(int batchsize) {
         if (batchsize == 0) {
-            log.warn("Batch Cleanup ARM Response Files - Batch size is 0, so not running");
+            log.warn("{}: {}: Batch Cleanup ARM Response Files - Batch size is 0, so not running", manifestFilePrefix);
```

Comment:
Changing the log messages around will impact the test teams automated tests. You will need to check and see if they will be impacted by your changes.

Reply:
Will let them know before we merge :)